ambry-api/src/main/java/com/github/ambry/account/AccountCollectionSerde.java

```diff
@@ -99,4 +141,25 @@ public static JSONObject containersToJson(Collection<Container> containers) {
       return containers;
     }
   }
+
+  /**
+   * Deserialize a collection of {@link Container}s in json from given InputStream.
+   * @param inputStream the {@link InputStream} that contains serialized json bytes.
+   * @param accountId the account id for these containers.
+   * @return a {@link Collection} of {@link Container}s.
+   */
+  public static Collection<Container> containersFromInputStreamInJson(InputStream inputStream, short accountId)
+      throws IOException {
```

Comment:
NIT: IOException won't happen here. We can remove it.

Reply:
ObjectMapper.readValue should still throw out IOException. We have to keep it.